tests/dm_control/test_dm_control_policy.py

```diff
@@ -0,0 +1,57 @@
+import gc
+
+from dm_control.suite import ALL_TASKS
+from nose2 import tools
+
+from garage.baselines import LinearFeatureBaseline
+from garage.envs.dm_control import DmControlEnv
+from garage.tf.algos import TRPO
+from garage.tf.envs import TfEnv
+from garage.tf.policies import CategoricalMLPPolicy, GaussianMLPPolicy
+from garage.tf.spaces import Box
+from tests.fixtures import GarageTestCase
+
+all_tasks_list = []
+for task in ALL_TASKS[0:1]:
+    all_tasks_list.append([task[0], task[1]])
+
+
+class TestDmControlPolicy(GarageTestCase):
+    @tools.params(*all_tasks_list)
```

Comment:
This many integration tests is unnecessary.

Reply:
This only runs one dm_control environment.

Reply:
`@tools.params(*all_task_lists)` implies that this runs a test case for every element of `all_tasks_list`. Are you saying `all_tasks_list` has only a single element?

Reply:
No, I was going to replace this code with all environments. I simply limited it to a single element for the purposes of the CI.

Reply:
The CI memory problems occur when testing a single environment against a single algorithm? (!)

Reply:
No, it's due to the other file that's testing all dm_control environments.

Reply:
Basically I'm making sure that both implementations are running in the CI, but since we don't need to test all dm_control environments, some renaming / refactoring is in order.

Reply:
...but this test is virtually identical to the other one. Does that mean that the memory leak happens with GarageTestCase but not with your custom test here?

Reply:
Both are experiencing memory leaks, and it's due to the same core problem of needing to create new TensorFlow graphs.